packages/lib/server-side/src/auth.ts

```diff
@@ -96,8 +96,13 @@ export const Auth = (getAccount, serverSecret: string) => {
 
             res.json(jwt);
         } catch (e) {
-            next(e);
             console.error('unable to create new session token ', e);
+            if (e instanceof Error && e.message === 'Signature invalid') {
```

Comment:
i think we could return every error with code 400

     return res.status(400).json({
                    error: e.message,
                });